types/indexer.go

```diff
@@ -25,6 +26,10 @@ var (
 )
 
 const (
+	// Transaction(tx) Block height index key.
+	//
+	// Note: Block height is cast as int when querying using the tx height index due to the lexicographic encoder library,
+	// this is safe because v0 block height should never exceed 2^63-1 on 64-bit systems or 2^31-1 on 32-bit systems.
```

Comment:
NIT (not actually): Block height should never exceed `2^31-1` on any 32+ bit system ;)